pkg/apis/vpc/v1alpha1/subnet_types.go

```diff
@@ -72,6 +76,7 @@ type SubnetList struct {
 // DHCPConfig is DHCP configuration.
 type DHCPConfig struct {
 	// +kubebuilder:default:=false
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
```

Comment:
Just one more comment, do we need put this kubebuilder CEL validation for `EnableDHCP` only or we should put on the whole struct: `DHCPConfig`

Reply:
Though DHCPConfig only has EnableDHCP for now, let's put the validation on the whole DHCPConfig. Thanks!